ci/do_ci.sh

```diff
@@ -90,9 +90,12 @@ CI_TARGET=$1
 
 if [[ $# -gt 1 ]]; then
   shift
-  TEST_TARGETS=$*
+  COVERAGE_TEST_TARGETS=$*
+  TEST_TARGETS="$COVERAGE_TEST_TARGETS"
 else
-  TEST_TARGETS=//test/...
+  # Coverage test will add QUICHE tests by itself.
+  COVERAGE_TEST_TARGETS=//test/...
```

Comment:
Do you intended to exclude quiche from coverage?

Reply:
Some of it has been added to coverage already: https://github.com/envoyproxy/envoy/blob/af6d1d460e5f05fd4e2dbae3777e076c3f381393/test/coverage/gen_build.sh#L38

Reply:
Update there to use the test_suite?

Reply:
the test_suite is not tagged with 'coverage_test_lib':
$ bazel query "attr('tags', 'coverage_test_lib', '@com_googlesource_quiche//:ci_tests')" emits empty results